package.json

```
@@ -26,14 +26,15 @@
     "test:bdd": "jest --config './jest.bdd.config.js' --testMatch '**/test/step-definitions/**/*.step.ts'",
     "test:coverage": "jest --coverage --coverageThreshold='{}'",
     "test:coverage-check": "jest --coverage",
-    "test:integration": "echo \"test:integration - TODO...\"",
+    "test:integration": "jest --runInBand --testMatch '**/test/integration/**/*.(test|spec).ts'",
```

```suggestion
    "test:integration": "jest --testMatch '**/test/integration/**/*.(test|spec).ts'",
```
why we want to run integration tests in sequence?